src/util/std_code.h

```diff
@@ -101,21 +101,39 @@ inline codet &to_code(exprt &expr)
 class code_blockt:public codet
 {
 public:
+  typedef std::vector<codet> code_operandst;
+
+  code_operandst &code_operands()
```

Comment:
:question: What's the purpose of having both `code_operands()` and `statements()`?

Reply:
The idea is to eventually have code_operands() in codet, once codet stops inheriting from exprt.